app/konnect/getting-started/migration.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Migrating from Kong Gateway On-Premises to Kong Konnect
+---
+
+The success of {{site.base_gateway}} can be attributed to its flexible deployment model, 
+exceptional performance, broad extensibility, and Open Source and Enterprise licensing models.
+This has resulted in a large deployment base over a variety of environments and topologies. 
+
+As organizations grow and scale they need for more advanced capabilities while lowering 
+operational complexity. Features such as strong [multi-tenancy](https://konghq.com/blog/enterprise/multi-tenancy), 
+federated API management, advanced security integrations, and others. With {{site.konnect_product_name}}, 
+users have access to a full featured API management platform which provides managed components and 
+builds on the lessons learned from deep customer usage of {{site.base_gateway}} and API management best practices. 
```

Comment:
```suggestion
Kong Konnect offers advanced features such as strong [multi-tenancy](https://konghq.com/blog/enterprise/multi-tenancy), 
federated API management, advanced security integrations. Migrating to Konnect from Kong Gateway is a straight forward process, designed to ensure a smooth migration experience.
```

The introduction distracts me from the purpose of the document which I believe to be: 
1. Konnect offers features that would make a user want to migrate from On-prem to Konnect
2. Migrating to Konnect is straight forward.